src/boundary_conditions.jl

```diff
@@ -292,49 +374,47 @@ end
 @inline apply_z_bottom_bc!(Gc, ::NFBC, args...) = nothing
 
 """
-    apply_z_top_bc!(Gc, top_bc, i, j, grid, boundary_condition_args...)
+    apply_z_top_bc!(Gc, top_flux::BC{<:Flux}, i, j, grid, args...)
 
-Add the part of flux divergence associated with a top boundary condition on c.
+Add the part of flux divergence associated with a top boundary condition on `c`.
 Note that because
 
-        tendency = ∂c/∂t = Gc = - ∇ ⋅ flux
+    ``tendency = ∂c/∂t = Gc = - ∇ ⋅ flux``
```

Comment:
This should be LaTex if wrapped in double quotes:

``\partial_t c = G_c = - \bnabla \cdot \bm{q}_c``,

where ``\bm{q}_c`` is the flux of ``c``.